CITATION.cff

```diff
@@ -37,13 +37,15 @@ authors:
   given-names: Huong
 - family-names: Okwa
   given-names: Nero
+- family-names: Rodríguez
```

Comment:
While you're at it, you might also remove Cvetanka and Lim.

Reply:
Done!